sql/migrations/0001_initial.sql

```diff
@@ -68,3 +68,16 @@ CREATE TABLE beacons
     epoch  INT NOT NULL PRIMARY KEY,
     beacon CHAR(4)
 ) WITHOUT ROWID;
+
+CREATE TABLE atxs
+(
+    id        CHAR(32) PRIMARY KEY,
+    layer     INT NOT NULL,
+    epoch     INT NOT NULL,
+    smesher   CHAR(64),
```

Comment:
please use same type as in identities table

Reply:
why not the same as in rewards?